L/LLVM/common.jl

```diff
@@ -225,7 +225,7 @@ CMAKE_FLAGS+=(-DCMAKE_CROSSCOMPILING=True)
 
 # Julia expects the produced LLVM tools to be installed into tools and not bin
 # We can't simply move bin to tools since on MingW64 it will also contain the shlib.
-CMAKE_FLAGS+=(-DLLVM_TOOLS_INSTALL_DIR="tools")
+CMAKE_FLAGS+=(-DCMAKE_INSTALL_BINDIR="tools")
```

Comment:
Is this change backwards compatible with older versions of llvm? Or do we need to protect it in a version guard?